#!/usr/bin/python
# -*- coding: utf-8 -*-

import sys
from BeautifulSoup import BeautifulSoup

soup = BeautifulSoup("\n".join(sys.stdin.readlines()))

sect = soup.find("section", { "id" : "skd" })

trs = sect.findAll("tr")

try:
    for tr in trs:
        trp = tr.findAll("td")
        a = tr.findAll("a")
        for idx, trp_ in enumerate(trp):
            trpu = trp_.string
            if idx == 1: print trpu.encode('utf-8') + "\t",
            if idx == 2: print trpu.encode('utf-8')
except IOError, e:
    pass

# SOME TOTALLY WEIRD SHIT WITH THESE LISTS
# LIST HAD LENGTH 1 and a[0] or any other we tried returned index error??!?!?
#    b = [ a ]#[0]#['href']               
#    print type( a[0]['href'] )